Documentation/network/standalone.rst

```diff
@@ -0,0 +1,1610 @@
+     --bpf-lb-mode=dsr \\
+     --bpf-lb-dsr-dispatch=ipip \\
+     --bpf-lb-rss-ipv4-src-cidr=192.168.0.0/16 \\
+     --bpf-lb-rss-ipv6-src-cidr=fd00::/96 \\
+     --devices=bond0
+
+In this example, the outer source IPv4 contains a ``192.168.0.0/16`` prefix and the last
+two quads are populated based on the hash of the inner packet. Similarly for IPv6, the
+source address holds a prefix of ``fd00::/96`` where the remaining 32 bits are populated
+based on the hash of the inner packet. The static prefix vs dynamic number of bits can be
+selected flexibly in order to accommodate for ACLs in the underlying network.
+
+Path MTU Discovery
+~~~~~~~~~~~~~~~~~~
+
+Given the IPIP/IP6IP6 encapsulation reduces the available MTU from the load-balancer to
+the node with the backend, Cilium supports client-side PMTU discovery. Meaning, the
+load-balancer responds with an IPv4 ICMP ``destination unreachable`` message with sub-type
+``fragmentation needed``, so that clients are able to cache this path information and
+to adjust their packet sizes for future transmissions. The IPv6 counterpart emits an
+ICMPv6 ``Packet Too Big`` message back to the sender. Both is auto-enabled under XDP
+mode.
+
+Maglev Consistent Hashing
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Cilium's eBPF load-balancer supports consistent hashing by implementing a variant
+of `the Maglev paper <https://storage.googleapis.com/pub-tools-public-publication-data/pdf/44824.pdf>`_
+hashing for backend selection. This option is selected through ``--bpf-lb-algorithm=maglev``
+and is in contrast to the default ``--bpf-lb-algorithm=random`` setting, which is picking
+a random backend for a new connection.
+
+Maglev improves resiliency in case of failures and provides better load-balancing
+properties as adding more load-balancers to a load-balancer group will make consistent
+backend selection throughout the group for a given 5-tuple without having to
+synchronize state with each group member. Therefore it is in particular suited
+for handling inbound north-south traffic with ECMP-based load-balancing in front.
+
+Similarly, upon backend removal the Maglev backend lookup tables are reprogrammed with
+minimal disruption for unrelated backends. For example, depending on the configuration,
+there would only be at most 1% difference in the reassignments for the given service.
+
+The ``--bpf-lb-maglev-hash-seed`` option is recommended to be set in order for Cilium
+to not rely on the fixed built-in seed. The seed is a base64-encoded 12 byte-random
+number, and can be generated once through ``head -c12 /dev/urandom | base64 -w0``,
+for example. If you have a group of load-balancers which all share the same set of
+services and backends, then every instance in that group must use the same hash
+seed for Maglev to work. Small example generated once which is later used for the
+subsequent Cilium configuration:
+
+.. parsed-literal::
+
+   SEED=$(head -c12 /dev/urandom | base64 -w0)
+   echo $SEED
```

Comment:
nit, we have the debate back and forth sometimes, but for a case like this I think it's helpful to get the `shell-session` highlight for the `SEED` and `echo` lines (since those are the ones you type into the terminal) then observe the output of the commands. Like so:
```suggestion
.. code-block:: shell-session
   $ SEED=$(head -c12 /dev/urandom | base64 -w0)
   $ echo $SEED
```